tests/test_cli.py

```diff
@@ -130,10 +130,12 @@ def test_compare_tool(self) -> None:
         ])
         UtilFunctions.run_compare_tool(ecu_variants=["somersault_lazy", "somersault_assiduous"])
 
-    @unittest.skipIf(import_failed, "import of PyInquirer failed")
+    @unittest.skipIf(browse_import_failed, "importing the browse tool failed")
     def test_browse_tool(self) -> None:
         browse_args = Namespace(pdx_file="./examples/somersault.pdx")
-        browse.run(browse_args)
+        with self.assertRaises(SystemError):
+            # browse can only be run interactively
+            browse.run(browse_args)
```

Comment:
Maybe mock the interactive part for testing?

Reply:
good idea. I tested to implement this, but the assumption that `sys.stdout` is a tty is burried too deeply within PyInquirer. (there are plenty of asserts and probably quite a few implicit assumptions...)

That said, I've noticed that this test stopped to ask for user input when PyInquirer is available and it is run in "bare metal mode" (without using `pytest`), so I mocked `sys.stdout.isatty()` to make it reliably bail out...